developer_guides/debugability/logger/index.rst

```diff
@@ -36,6 +36,7 @@ Usage sof-logger <option(s)> <file(s)>
 -v ver_file	Enable checking the firmware version with the ver_file file instead of the default: "/sys/kernel/debug/sof/fw_version"
 -s		Take a snapshot of state
 -r		Less formatted output for chained log processors
```

Comment:
The commit description doesn't make sense for me. I think it is better to use the commit description to explain in detail what is the purpose of '-L'

Reply:
updated